api/envoy/api/v2/route/route_components.proto

```diff
@@ -137,9 +137,12 @@ message VirtualHost {
 
   // Decides whether the :ref:`x-envoy-attempt-count
   // <config_http_filters_router_x-envoy-attempt-count>` header should be included
-  // in the upstream request. Setting this option will cause it to override any existing header
+  // in the upstream request and the downstream response.
+  // Setting this option will cause it to override any existing header
   // value, so in the case of two Envoys on the request path with this option enabled, the upstream
-  // will see the attempt count as perceived by the second Envoy. Defaults to false.
+  // will see the attempt count as perceived by the second Envoy. Similarly, if there are multiple Envoys
+  // on the response path with this option enabled, the downstream will see the attempt count
+  // as perceived by the first Envoy in the path. This field Defaults to false.
```

Comment:
How about a regression test for the overwrite?

also 
field Defaults -> field defaults